src/_posts/platform/getting-started/2000-01-01-free-trial.md

```diff
@@ -16,19 +16,21 @@ have to give us your billing details nor payment methods.
 ### What can I do under the free trial?
 
 You can have a **maximum of 1 application** using a **maximum of 5 containers**
-(S or M size). You can add as many addons you want if they have free plans
-available.
+(S or M size).
+
+You can add as many addons you want if they have free plans available.
 
 ### How to break out of the free trial?
 
-Enter a payment method on the [web dashboard](https://my.scalingo.com).
+Enter a payment method on the [Web Dashboard](https://dashboard.scalingo.com).
```

Comment:
Why upper case `W` and `D` to the words "Web Dashboard"? 🤔  Aren't they just common name?